src/SFA.DAS.ApplyService.Database/Tables/Apply.sql

```diff
@@ -15,7 +15,13 @@
     [UpdatedBy] NVARCHAR(256) NULL, 
     [DeletedAt] DATETIME2 NULL, 
     [DeletedBy] NVARCHAR(256) NULL,
-    [UKPRN] AS JSON_VALUE(ApplyData, '$.ApplyDetails.UKPRN') PERSISTED,
+    [UKPRN] AS JSON_VALUE(ApplyData, '$.ApplyDetails.UKPRN') PERSISTED, 
+    [Assessor1UserId] NVARCHAR(256) NULL, 
+    [Assessor2UserId] NVARCHAR(256) NULL, 
+    [Assessor1Name] NVARCHAR(256) NULL, 
+    [Assessor2Name] NVARCHAR(256) NULL, 
+    [Assessor1ReviewStatus] NVARCHAR(20) NULL, 
+    [Assessor2ReviewStatus] NVARCHAR(20) NULL
```

Comment:
Can the schema change be put into a separate PR so it can be merged into master early

Reply:
I think we're going to need to create a submaster for this anyway as my next piece of work depends on more than just the sql changes.